config_generator.py

```diff
@@ -5,6 +5,7 @@
 import requests
 from pathlib import Path
 import os
+import imp
```

Comment:
Don't use this module. This one is [deprecated](https://docs.python.org/3/library/imp.html).